packages/design-system/src/components/Separator/Separator.tsx

```diff
@@ -34,6 +33,47 @@ const Separator = styled(SeparatorPrimitive.Root, {
           height: '$7',
         },
       },
+      full: {
+        '&[data-orientation="horizontal"]': {
+          height: '1px',
+          width: '100%',
+        },
+
+        '&[data-orientation="vertical"]': {
+          width: '1px',
+          height: '100%',
+        },
+      },
+    },
+    margin: {
+      '0': {},
```

Comment:
Let us remove this later.

Also, this probably a bit away from the intent of `radix` primitives w/ `stitches` and being able to pass `css`.

Creating app-specific Separators may be a good compromise, but not needed here potentially.